package/unix/mucommander.sh

```diff
@@ -32,6 +32,13 @@ else
   BASE_FOLDER=$0
 fi
 
+if [ "$(uname)" = "Darwin" ]; then
+    EXTRA_OPTIONS='--add-exports java.desktop/com.apple.eawt=ALL-UNNAMED
+      --add-exports java.desktop/com.apple.eio=ALL-UNNAMED
+      --add-exports java.desktop/com.apple.laf=ALL-UNNAMED'
+    echo $EXTRA_OPTIONS
```

Comment:
if this is intended for debugging, please file some context of what is being echoed

Reply:
my bad! removed this line